handbook/company/pricing-features-table.yml

```diff
@@ -742,6 +745,67 @@
+#  ╚╗╔╝║ ║║  ║║║║╣ ╠╦╝╠═╣╠╩╗║║  ║ ║ ╚╦╝  ╚═╗║  ║ ║╠╦╝║╣ ╚═╗  ───  ║╣ ╠═╝╚═╗╚═╗  ╠═╣║║║ ║║  ║  ╚╗╔╝╚═╗╚═╗
+#   ╚╝ ╚═╝╩═╝╝╚╝╚═╝╩╚═╩ ╩╚═╝╩╩═╝╩ ╩  ╩   ╚═╝╚═╝╚═╝╩╚═╚═╝╚═╝       ╚═╝╩  ╚═╝╚═╝  ╩ ╩╝╚╝═╩╝  ╚═╝ ╚╝ ╚═╝╚═╝
+- industryName: Vulnerability scores (EPSS and CVSS) 
+  tier: Premium
+  usualDepartment: Security
+  productCategories: [Vulnerability management]
+  pricingTableCategories: [Vulnerability management]
+  buzzwords: [Risk scores,Cyber risk,Risk reduction,Security operations effectiveness,Peer benchmarking,Security program effectiveness,Risk-based exposure scoring,Threat context,Cyber exposure,Exposure quantification and benchmarking,Optimize security investments,Vulnerability assessment]
+  demos:
+    - description: Fleet enables a more modern, threat-first prioritization approach to vulnerability management.
+      quote: In reality, across our inventory of devices, it's unlikely to ever be exploited. I'd rather do that legwork on my team and then go and ask and prioritize work on these infrastructure teams that are already busy with things that could or could not be vulnerable. Being able to be more exact allows us to go to these teams less, which saves everybody time.
+      moreInfoUrl: https://www.youtube.com/watch?v=G5Ry_vQPaYc&t=131s
+  waysToUse:
+    - description: By leveraging EPSS (Exploit Prediction Scoring System), security professionals gain insight on the true risk behind rated CVEs. 
+    - description: An Introduction to EPSS, The Exploit Prediction Scoring System
+    - moreInfoUrl: https://www.youtube.com/watch?v=vw1RlZCSRcQ
+#  ╔═╗╦╔═╗╔═╗  ╦╔═╔═╗╦  ╦╔═╗
+#  ║  ║╚═╗╠═╣  ╠╩╗║╣ ╚╗╔╝╚═╗
+#  ╚═╝╩╚═╝╩ ╩  ╩ ╩╚═╝ ╚╝ ╚═╝
+- industryName: CISA KEVs (known exploited vulnerabilities) 
+  tier: Premium
+  usualDepartment: Security
+  productCategories: [Vulnerability management]
+  pricingTableCategories: [Vulnerability management]
+  demos:
+    - description: 
+      moreInfoUrl:
+  waysToUse:
+    - description: Help teams work on vulnerabilities that have actually been exploited (CISA KEVs) or have a high probability of being exploited (EPSS), or whatever is important in your environment.
+    - description: Using CISA KEV for Vulnerability Management
+    - moreInfoUrl: https://www.youtube.com/watch?v=Z3mw2oxssYk
+#  ╔═╗╔═╗╔╦╗╔═╗╦ ╦╔═╗╔╦╗  ╦  ╦╔═╗╦═╗╔═╗╦╔═╗╔╗╔
+#  ╠═╝╠═╣ ║ ║  ╠═╣║╣  ║║  ╚╗╔╝║╣ ╠╦╝╚═╗║║ ║║║║
+#  ╩  ╩ ╩ ╩ ╚═╝╩ ╩╚═╝═╩╝   ╚╝ ╚═╝╩╚═╚═╝╩╚═╝╝╚╝
+- industryName: Patched version 
+  tier: Premium
+  comingSoonOn: #exists in vulnerability dashboard, but tracked for product in https://github.com/fleetdm/fleet/issues/15919
+  usualDepartment: Security
+  productCategories: [Vulnerability management]
+  pricingTableCategories: [Vulnerability management]
+  demos:
+    - description: 
+      moreInfoUrl:
+  waysToUse:
+    - description: By extracting metadata from the National Vulnerability Database (NVD), we can determine which version of software is no longer vulnerable. 
```

Comment:
Mike: Your changes here make good sense. Should we merge this "waysToUse" into the existing Risk-based vuln management feature? Then we can just delete this "Patched version" thing altogether.